modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -116,4 +118,22 @@ object SeqexecEngine {
 
       seq.steps.map(viewStep)
     }
+
+    // Configuration stuff
+
+
+  def seqexecConfigure: Kleisli[Task, Config, Unit] = Kleisli { cfg: Config => {
+      val odbHost = cfg.require[String]("seqexec-engine.odb")
+      val dhsServer = cfg.require[String]("seqexec-engine.dhsServer")
+      val dhsSim = cfg.require[Boolean]("seqexec-engine.dhsSim")
+      val tcsSim = cfg.require[Boolean]("seqexec-engine.tcsSim")
+      val instSim = cfg.require[Boolean]("seqexec-engine.instSim")
+      val gcalSim = cfg.require[Boolean]("seqexec-engine.gcalSim")
+
+      Task.delay(ODBProxy.host(new Peer(odbHost, 8443, null))) *>
```

Comment:
I guess this could be something like new `SeqTranslate(setting, dhsClient, odbProxy)` and return it on the `Kleisli`